modules/kg-schemas/src/main/resources/schemas/nexus/core/resource/v0.1.0.json

```diff
@@ -0,0 +1,111 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    {
+      "nxcv": "https://bbp-nexus.epfl.ch/vocabs/nexus/core/v0.0.1/",
+      "this": "{{base}}/schemas/nexus/core/resource/v0.0.1/shapes/"
+    }
+  ],
+  "shapes": [
+    {
+      "@id": "this:ResourceShape",
+      "@type": "sh:NodeShape",
+      "and": [
+        {
+          "@id": "this:RevisionShape",
+          "@type": "sh:NodeShape",
+          "targetObjectsOf": "nxcv:rev",
+          "description": "The revision number of the instance",
+          "datatype": "xsd:integer",
+          "minInclusive": 1
+        },
+        {
+          "@id": "this:DeprecatedShape",
+          "@type": "sh:NodeShape",
+          "targetObjectsOf": "nxcv:deprecated",
+          "description": "A boolean tag indicating whether the resource is deprecated or not.",
+          "datatype": "xsd:boolean"
+        },
+        {
+          "@id": "this:PublishShape",
+          "@type": "sh:NodeShape",
+          "targetObjectsOf": "nxcv:published",
+          "description": "A boolean tag indicating whether the resource has been published or not.",
+          "datatype": "xsd:boolean"
+        },
+        {
+          "@id": "this:LinkShape",
```

Comment:
I wonder if we should externalize HATEOAS specific constraints as it's shared between resources and listing / search. Thoughts?